bin/usi

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
```

Comment:
I know it's weird and a bit off, but for now I think this should go in cmd/ and not bin/ -- simply because it's most similar to things like cmd/faux which do the same thing (start container).  At some point we have to clean everything up so it makes sense, but for now I'm going with consistency...